src/removeStopwords.js

```diff
@@ -0,0 +1,38 @@
+// SUGGESTIONS: refacture ino shouldDisplay
+// and this function take a word and the language, then it returns true or false, and this function should be used wherever you are trying to display the words
+
+import {  STOPWORDS } from './constants';
+
+const filterWords = (wordCountObject, wordsToFilterOut) => {
+  for (var key in wordCountObject) {
+    var containsWord = (wordsToFilterOut.indexOf(key) > -1);
+    if (containsWord) {
+      delete wordCountObject[key];
```

Comment:
as discussed, this is risky as it mutates the original object.

Reply:
see comment above. Should be fine if this is a separate object.